themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -28,15 +28,28 @@
       <? foreach (($top = $results->getRecommendations('top')) as $current): ?>
         <?=$this->recommend($current)?>
       <? endforeach; ?>
-      <?=$this->transEsc("Showing")?>
-      <strong><?=$this->localizedNumber($results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($results->getEndRecord())?></strong>
-      <? if (!isset($view->skipTotalCount)): ?>
-        <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
+      <?
+        $transParams = [
```

Comment:
I am still seeing the problem:

![image](https://cloud.githubusercontent.com/assets/309069/23564170/947a730a-0017-11e7-942a-4f3676f76821.png)